app/src/reducers/coverPage.js

```diff
@@ -0,0 +1,11 @@
+const initialState = null;
+import { GET_COVER_PAGE_ENTRIES } from '../actions';
+
+export default function (state = initialState, action) {
+  switch (action.type) {
+    case GET_COVER_PAGE_ENTRIES:
+      return action.payload;
```

Comment:
The actual requirement is that a reducer should never return a modified version of the previous state object, but should always return a fully new state object. That's why we use Object.assign, to create a copy of the previous state. we are achieving the same here.